src/disk_interface.cc

```diff
@@ -60,11 +60,11 @@ int MakeDir(const string& path) {
 TimeStamp TimeStampFromFileTime(const FILETIME& filetime) {
   // FILETIME is in 100-nanosecond increments since the Windows epoch.
   // We don't much care about epoch correctness but we do want the
-  // resulting value to fit in an integer.
+  // resulting value to fit in a 64-bit integer.
```

Comment:
https://msdn.microsoft.com/en-us/library/windows/desktop/ms724284(v=vs.85).aspx
indicates that perhaps the raw time is just a 64-bit integer, so perhaps we could just use the time directly.  Another way of saying this is that we could drop lines 66 and 67 I think (and then adjust the relevant comments).

Oh this doc is better:
https://msdn.microsoft.com/en-us/library/windows/desktop/ms724290(v=vs.85).aspx